@here/harp-examples/webpack.config.js

```diff
@@ -195,9 +198,23 @@ const assets = [
         toType: "dir"
     },
     require.resolve("three/build/three.min.js"),
-    resolveOptional("@here/harp.gl/dist/harp.js", "bundle examples require `yarn build-bundle`"),
-    resolveOptional("@here/harp.gl/dist/harp-decoders.js")
-].filter(asset => asset); // ignore stuff that is not found
+    {
+        from: resolveOptional(`@here/harp.gl/dist/harp${harpBundleSuffix}.js`, "bundle examples require `yarn build-bundle`"),
+        to: "harp.js"
+    },
+    {
+        from: resolveOptional(`@here/harp.gl/dist/harp-decoders${harpBundleSuffix}.js`),
+        to: "harp-decoders.js"
+    }
+].filter(asset => {
+    if (asset === undefined || asset === null) {
+        return false;
+    } else if (typeof asset === "string") {
+        return true;
+    } else if (typeof asset === "object") {
+        return asset.from;
```

Comment:
Just a blind shot.... do not know the topic well, but maybe we should check asset.from and asset.to, both for undefined, but also if it is empty - if ever possible.